fbgemm_gpu/codegen/embedding_backward_split_host_template.cpp

```diff
@@ -434,17 +434,6 @@ class {{ autograd_func }} :
     auto& grad_output = grad_outputs[0];
     {%- endif %}
 
-    // FIXME: to support aligned memory access in Vec4T load/store function
-    // 16 for FP32 and 8 for FP16
-    if (grad_output.dim() > 1 &&
-        (reinterpret_cast<uint64_t>(grad_output.data_ptr()) % 16 != 0 ||
-        grad_output.stride(1) != 1 || grad_output.stride(0) % 4 != 0)) {
-        grad_output = grad_output.contiguous();
-    }
-    if (reinterpret_cast<uint64_t>(grad_output.data_ptr()) % 16 != 0) {
-        grad_output = at::empty_like(grad_output).copy_(grad_output);
-    }
```

Comment:
Intentional?

Reply:
I moved this into the cuda kernels, I can't have that data_ptr with meta tensors. Do you have a suggestion on how to do this?